### PR TITLE
Authorize batch actions

### PR DESCRIPTION
**Pull request by Linuus**
# PR 100

Fixes: #33 

What do we want the notice to say?
Should the notice be localized under `flash` or `batch_action`? :)
Different flash translations for different batch actions?

### CODE REVIEW

File: lib/godmin/resources/resource_controller.rb [hunk 1]
```diff
@@ -154,9 +154,15 @@ def perform_batch_action
         return false unless params[:batch_action].present?
 
         item_ids = params[:id].split(",").map(&:to_i)
+        records = @resource_class.find(item_ids)
+        records = records.select { |r| policy(r).send("batch_action_#{params[:batch_action]}?") }
```
Comment (jensljungblad) [inline lib/godmin/resources/resource_controller.rb]: check if authorization enabled

**Comment by Linuus** [comment]
I fixed your comments. What about the flash message? Any suggestions? :)

Summoning @jensljungblad

**Comment by jensljungblad** [comment]
CHANGELOG
README

File: test/lib/godmin/resources/resource_service/batch_actions_test.rb [hunk 2]
```diff
@@ -4,13 +4,7 @@ module Godmin
...
     class BatchActionsTest < ActiveSupport::TestCase
       def setup
-        resource_class = Class.new do
-          def self.find(ids)
-            ids
-          end
-        end
-
-        @article_service = ArticleService.new(resource_class: resource_class)
+        @article_service = ArticleService.new(resource_class: Class.new)
```
Comment (jensljungblad) [inline test/lib/godmin/resources/resource_service/batch_actions_test.rb]: can it be removed :question:

**Pull request merged**